### DISCUSSION TITLE
I added a Coral PCIe detector, not what to do with USB Coral and GPU?[Detector Support]:

### DISCUSSION THREAD

**Discussion by TheeDudenator** [blakeblackshear/frigate#13589] [Detector Support]
### Describe the problem you are having

I have 9 4k cameras
I do have the second feed from each camera downsized for detection.

Should I just unplug and sell the USB coral?  Is it really helping now?
Do I still need this GPU configured?

Everything is working fine, just wondering what is optimal.

### Version

0.14.1

### Frigate config file

```yaml
NA
```

### docker-compose file or Docker CLI command

```yaml
NA
```

### Relevant Frigate log output

```shell
NA
```

### Relevant go2rtc log output

```shell
NA
```

### Operating system

Debian

### Install method

Docker Compose

### Object Detector

Coral

### Screenshots of the Frigate UI's System metrics pages

![Detectors](https://github.com/user-attachments/assets/9696e8d5-e2fc-4b19-9a2d-5e0dea467273)

**Comment by NickM-27** [comment]
you can't use amd for detection currently, so you need both. You could get rid of the usb if you wanted to or keep using it in case there is a lot of activity on many cameras at the same time

**Comment by TheeDudenator** [comment]
What is the reason/usage of AMD-vaapi then?
Should I have it?

Or is that how Frigate works?

**Reply by NickM-27** [reply]
it uses it to decode the video streams
